core/common/src/main/java/alluxio/underfs/options/CreateOptions.java

```diff
@@ -60,6 +64,7 @@ private CreateOptions(String authUmask) {
     mOwner = null;
     mGroup = null;
     mMode = ModeUtils.applyFileUMask(Mode.defaults(), authUmask);
+    mXAttr = new HashMap<>();
```

Comment:
Ah I missed the constructor here. I was trying to set the default size of the map to `1` so that a minimal amount of memory is used. I can make this `Nullable` similar to the others though. I'm just wary of NPEs